.github/workflows/release-beta.yml

```
@@ -0,0 +1,82 @@
+name: Release 3x Beta
+
+on:
+  pull_request:
+#  push:
+    branches:
+      - feat_v3.x
+  workflow_dispatch:
+
+jobs:
+  release-3.x-beta:
```

actionlint (1.7.4)

invalid job ID "release-3.x-beta". job ID must start with a letter or _ and contain only alphanumeric characters, -, or _

(id)